tensorflow/compiler/xla/service/gpu/gpu_compiler.cc

```diff
@@ -444,6 +444,9 @@ Status GpuCompiler::OptimizeHloModule(
       // bitcast(bitcast) with one bitcast. This leads to having to
       // linearize and then delinearize the index.
       options.set_replace_transpose_with_bitcast(false);
+#if TENSORFLOW_USE_ROCM
```

Comment:
Could we make it a runtime check somehow?

Reply:
OK - checked the platform string instead. Is that better? :)